plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/VeracodeReader.java

```diff
@@ -128,9 +128,11 @@ private TestCaseResult parseVeracodeVulnerability(Node flaw) {
 
         String testcase = getAttributeValue("sourcefile", flaw);
         if (testcase.startsWith(BenchmarkScore.TESTCASENAME)) {
-            String testno =
-                    testcase.substring(BenchmarkScore.TESTCASENAME.length(), testcase.length() - 5);
-            tcr.setNumber(Integer.parseInt(testno));
+            // Suppose that remove the .html extension, to check
```

Comment:
Remove all commented out code if it's not needed anymore. This applies throughout this pull request.

Reply:
Done